ts/screens/messages/MessageDetailsScreen.tsx

```diff
@@ -0,0 +1,78 @@
+import { Body, Button, Container, Content, H1, Left, Text } from "native-base";
+import * as React from "react";
+import {
+  NavigationInjectedProps,
+  NavigationScreenProp,
+  NavigationState
+} from "react-navigation";
+import MessageDetailsComponent from "../../components/messages/MessageDetailsComponent";
+import AppHeader from "../../components/ui/AppHeader";
+import IconFont from "../../components/ui/IconFont";
+import I18n from "../../i18n";
+
+interface ParamTypeObject {
+  subject: string;
+  serviceOrganizationName: string;
+  serviceDepartmentName: string;
+  markdown: string;
+  serviceName: string;
+  createdAt: Date;
+}
+
+interface ParamType {
+  readonly details: ParamTypeObject;
+}
+
+interface StateParams extends NavigationState {
```

Comment:
why we need these 3 nested types? (StateParams -> ParamType -> ParamTypeObject)

Reply:
StateParams are the details message param passed from the message component:
details: {
              createdAt,
              markdown,
              serviceName,
              serviceDepartmentName,
              serviceOrganizationName,
              subject
}

I kept the same syntax as the pr already merged like (screens/wallet/TransactionsScreen.tsx)